practitioner-guides/organizational-participation.md

```diff
@@ -94,6 +94,7 @@ It can take time for projects to recruit users and contributors, so it is partic
 
 # Additional Reading
 
+* We have a [short video](https://www.youtube.com/watch?v=xTK2xsd4jZE&list=PL60k37cxI-HSHV4-rEsWMzExw2y2Oq79Z&index=4) (< 3 minutes) devoted to this guide on the CHAOSS YouTube channel.
```

Comment:
I think this link should be this: https://www.youtube.com/watch?v=mztEQMgam5Y&list=PL60k37cxI-HSHV4-rEsWMzExw2y2Oq79Z&index=4